src/field_theory/normal.lean

```diff
@@ -68,4 +68,22 @@ begin
     (minimal_polynomial.dvd_map_of_is_scalar_tower K hx)⟩,
 end
 
+variables {F} {E} {E' : Type*} [field E'] [algebra F E']
+
+lemma normal.of_alg_equiv (h : normal F E) (f : E ≃ₐ[F] E') : normal F E' :=
```

Comment:
It might be nice to add a second lemma
```lean
lemma normal.iff_alg_equiv (f : E ≃ₐ[F] E') : normal F E ↔ normal F E' :=
⟨λ h, h.of_alg_equiv f, λ h, h.of_alg_equiv f.symm⟩
```
since that can be used with `rw`. `alg_equiv.transfer_normal` or something might be another name for that.